Shared/UI/Views/VideoView.swift

```diff
@@ -35,50 +35,6 @@ class VideoView: BaseView {
     override func initializeSubviews() {
         super.initializeSubviews()
         
-        NotificationCenter.default.publisher(for: AVPlayer.rateDidChangeNotification)
-            .filter({ notification in
-                if let player = notification.object as? AVPlayer,
-                   player === self.playerLayer.player {
-                    return true
-                }
-                
-                return false
-            }).mainSink { [unowned self] (value) in
-                guard let player = value.object as? AVQueuePlayer else { return }
-                
-                switch player.status {
-                case .unknown:
-                    break
-                case .readyToPlay:
-                    switch player.timeControlStatus {
-                    case .paused, .waitingToPlayAtSpecifiedRate:
-  
-                        guard let reason = player.reasonForWaitingToPlay else { return }
-                                                
-                        switch reason {
-                        case .evaluatingBufferingRate:
-                            logDebug("evaluatingBufferingRate")
-                        case .toMinimizeStalls:
-                            logDebug("toMinimizeStalls")
-                        case .noItemToPlay:
-                            logDebug("noItemToPlay")
-                        default:
-                            logDebug("Unknown \(reason)")
-                        }
-
-                    case .playing:
-                        break
-                    @unknown default:
-                        break
-                    }
-                case .failed:
-                    break
-                @unknown default:
-                    break
-                }
-                
-            }.store(in: &self.cancellables)
-        
```

Comment:
I don't think we need this? Knowing about it is very helpful though.

Reply:
Yeah I was just using if for testing. Fine to remove it.